### ISSUE TITLE
Body/Head Scale Factor

### ISSUE THREAD

**Issue by TurtleShroom** [Vanilla-Expanded/VanillaExpandedFramework#104]
Good evening.

I am updating one of my Mods to V1.5 , and one error I am, now getting concerns the Gene Mod Extensions.

```
`Exception parsing <bodyScaleFactor>(0.981,2.5)</bodyScaleFactor> to type System.Nullable`1[System.Single]: 
System.ArgumentException: Exception parsing System.Single from "(0.981,2.5)" ---> System.FormatException:
 Input string was not in a correct format.`
```

I searched your documentation, but got nothing but a verbatim copy and paste in cryptic C-Sharp. What actually changed?

This is the code I am using:
```
	<modExtensions MayRequire="ludeon.rimworld.biotech">
		<li Class="VanillaGenesExpanded.GeneExtension" MayRequire="ludeon.rimworld.biotech">
			<bodyScaleFactor>(0.981,2.5)</bodyScaleFactor>
			<headScaleFactor>(1.07,1.072)</headScaleFactor>
		</li>
	</modExtensions>
```

Thank you for any assistance you may render.

Warm regards,
-TURTLESHROOM

**Comment by juanosarg** [comment]
We forgot to document the change! It is handled via a stat now. See the example in Hussars: https://github.com/Vanilla-Expanded/VanillaRacesExpanded-Hussar/blob/5352d16142bce69dd82098193030d412de19fbe7/1.5/Defs/GeneDefs/GeneDefs_Misc.xml#L66

**Comment by TurtleShroom** [comment]
> We forgot to document the change! It is handled via a stat now. See the example in Hussars: https://github.com/Vanilla-Expanded/VanillaRacesExpanded-Hussar/blob/5352d16142bce69dd82098193030d412de19fbe7/1.5/Defs/GeneDefs/GeneDefs_Misc.xml#L66

Thank you very much!

**Comment by TurtleShroom** [comment]
Thank you very much!

**Comment by TurtleShroom** [comment]
> We forgot to document the change! It is handled via a stat now. See the example in Hussars.

How do I make them taller, but not wider? I have a Gene that makes those who have it very lanky, but also very tall. The only code I see here appears to be something that scales a Pawn's width AND height at the same time.

**Comment by juanosarg** [comment]
I don't think that functionality is there anymore, as it caused problems with heads, iirc
